test/ui-test/testSuites/suite_status/tst_wallet/test.feature

```diff
@@ -0,0 +1,20 @@
+Feature: Status Desktop Wallet
```

Comment:
Maybe we can extend the test by checking the account balance, as requested in the task description

Reply:
account balance is somehow tricky, as account balance can change easily, I don't know an account with a static balance